pkg/servicetest/testdata/services/docker-compose.solr.yaml

```diff
@@ -71,7 +71,7 @@ services:
     entrypoint: [ "sh", "-c", "docker-entrypoint.sh solr-precreate dev /solr-conf" ]
 
     external_links:
-      - "ddev-router:${DDEV_SITENAME}.ddev.site"
+      - "ddev-router:${DDEV_SITENAME}.${DDEV_TLD}"
```

Comment:
Can I assume this is the only other place this surfaces?

Reply:
I tried to look carefully. That's just an example file, but I was surprised to see the "ddev.site" wired in there. I'll bet there are several more in ddev-contrib.